api/src/main/java/org/openmrs/api/impl/ProviderServiceImpl.java

```diff
@@ -257,4 +259,17 @@ public boolean isProviderIdentifierUnique(Provider provider) throws APIException
 	public Provider getProviderByIdentifier(String identifier) {
 		return dao.getProviderByIdentifier(identifier);
 	}
+	
+	@Override
+	public Provider createProviderFromUser(User user) {
+		if (user != null) {
+			Provider p = new Provider();
+			
+			p.setPerson(user.getPerson());
+			p.setIdentifier(user.getUsername());
+			if (Context.getProviderService().getProvidersByPerson(user.getPerson()).isEmpty())
```

Comment:
If clause statements need to be wrapped inside curly braces based on our convention